modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,213 @@
+[[hub-online-sync]]
+= Hub Online Synchronization
+
+
+//OM 2025-04-28: shall we write hub or HUb, peripheral or Peripheral? In the GUI, I saw mixed cases.
+
+== Introduction 
+
+Hub online synchronization reuses the existing repository syncing mechanism and synchronizes channels in the peripheral servers from the repositories on the hub server.
+
+Once the connection between hub and peripheral servers is established, the hub server becomes the main source of data for the peripheral server, effectively replacing {scclongform}.
+
+The main characteristics of this feature are:
+
+* There can only be one hub server per connection, with one or more peripheral servers.
+
+* Each peripheral server can only have one hub server. 
+
+* Content can be synchronized on regular basis, or on demand.
+
+
+
+== Registration of the hub and peripheral servers
+
+Hub online synchronization is configured from menu menu:Admin[Hub Configuration].
+
+Configuration process uses token which uniquely identifies peripheral server's connection to the hub.
+
+There are two ways to register a peripheral server to the hub server:
+
+. by using a combination of token creation on the peripheral, and subsequent registration on the hub server.
+  This method uses <<peripheral-token-generation>> and <<token-transfer>>.
+. by direct registration from the hub, without any interactions with the peripheral server. 
```

Comment:
```suggestion
. by direct registration from the hub, without any user interactions with the peripheral server. 
```

Maybe it's better to clarify it's without **user** interaction or the reader might think this registration method can be done fully on the hub without needing the peripheral to be up and reachable. With this method the hub is basically using API calls to perform the first steps of the token generation procedure.